notebooks/basics/single-systems.ipynb

```diff
@@ -624,13 +625,13 @@
    "source": [
     "### 2.1 Quantum state vectors <a id='single-systems-quantum-states'></a>\n",
     "\n",
-    "A *quantum state* of a system is represented by a column vector in a similar way to the representation of probabilistic states by vectors, including the assumption that the indices of the vector are placed in correspondence with the classical state set of the system being considered.\n",
+    "A *quantum state* of a system is represented by a column vector, similar to probabilistic states. As before, the indices of the vector label the classical states of the system.\n",
     "Vectors representing quantum states are characterized by these two properties:\n",
     "\n",
     "1. The entries of a quantum state vector are *complex numbers*.\n",
     "2. The sum of the *absolute values squared* of the entries of a quantum state vector must equal $1.$\n",
     "\n",
-    "Thus, in contrast to the probabilistic case, vectors representing quantum states need not have nonnegative real number entries, and it is the sum of the absolute values squared of the entries (as opposed to the sum of the entries) that must equal $1.$\n",
+    "Thus, in contrast to the probabilistic case, vectors representing quantum states need not have nonnegative real number entries, and it is the sum of the absolute values squared of the entries (as opposed to the sum of the entries) that must equal $1.$ Simple as this change may be, this is the essential difference between quantum mechanics and regular probability theories. Any speedup from a quantum computer, or improvement in a communication protocol, ultimately derives from this simple mathematical change. \n",
```

Comment:
These added sentences aren't quite working for me. A small issue is that the differences read as two changes, not one. More importantly, I don't want to say anything about quantum **mechanics**. (Maybe "... essential difference between quantum and classical information."?)

Reply:
These points make sense to me. Would you consider a rewording, or should we drop it? I hoped to emphasize that these simple definitional changes have a big impact, to emphasize there is nothing "more" to quantum information per se.